### ISSUE TITLE
Add logo to Nav bar

### ISSUE THREAD

**Issue by Feissit** [CUCentralAdvancement/essential-cu#342]
High Priority

Add [logo](https://www.dropbox.com/home/ADVMarComm%20Team%20Folder/01-CURRENT%20PROJECTS/2021/IR21/Web-Elements/Global?preview=IR21-Wordmark_CUcolor-notitle-stroke.png) to right side of nav bar: 

Default page width: 1440px

.ir-21-wordmark-c-ucolor-notitl { 
height: 93px; 
width: 188.82px; 
}

From right edge of screen: 48.03px
From left: 1203.15px
Flush with top of screen

<img width="831" alt="Screen Shot 2021-10-13 at 12 55 22 PM" src="https://user-images.githubusercontent.com/91230278/137195803-a91cd4fb-ba28-4388-9fe9-a513a61eaff6.png">

**Comment by duquennoisa** [comment]
@alexfinnarn Thanks for updating the logo. I think its best without the added 48px padding to the image and keep the div with 2rem padding on the left. Thoughts? 
![image](https://user-images.githubusercontent.com/70037329/137365448-686e4de5-7278-48cb-b9ef-2696daeb1736.png)

**Comment by alexfinnarn** [comment]
@duquennoisa Yes, I agree with the padding value, although then the logo on the left should probably have the same padding? 

I'd like to keep using Tailwind's spacing array unless another one is provided, but most of the time Tailwind spacing values are going to be in between what InVision reports on the designs. 

I'm also unclear if you want the header contents to expand full width. Last year's report set the header contents max-width to the "container". I don't think anything actually stretched the whole screen, except on smaller screen sizes. 

I know other issues have notes about breakpoints, but should the header fill the width of the container - "gutters"?

**Comment by alexfinnarn** [comment]
@duquennoisa I was hoping to use and keep the default spacing scale and use more exact/custom values for font, colors, shadows, etc. The spacing array is probably the most used "utility" in Tailwind and unless it's replaced with another scale than doing math like "the code has padding-right 4th position but people want it larger, I will use the 5th position" or using ratios between elements.

For your analogy, from my perspective, it's like you invite a guest over after you've done some cleaning but you can still see dust, clutter, and things that embarrass you. What does the guest say when they enter your house? "You have a beautiful house here." To them, they don't have any idea things are out of place. They came to your house for dinner anyway so let's eat.

I found out a while ago that this group doesn't really care about developer things like performance. I spent some time browsing last year's report in simulated slow-speed networks. This is important for people trying to read those stories in places where they don't have fancy smartphones and fast networks. I made sure images were optimized and other things too...but it did not matter. No one but me noticed or said anything. From then on, I tried to not think too much about performance or other dev things and focus on the core experience of the stakeholder.

So, to sum things up, I think if more exact measurements are required/desired, then there's no point continuing to update the code with default Tailwind values. And if that's the case, then the more exacting value QA should probably wait until content gets placed. You might want the sizes of elements to change with the cropping of pictures, I'm not sure...but I feel like last year a lot of attention was paid to that regard.
